src/randoop/JunitFileWriter.java

```diff
@@ -153,6 +143,19 @@ private File writeSubSuite(List<ExecutableSequence> sequencesForOneFile, String
     return file;
   }
 
+/*
+ * Writes a test method to the output stream for the sequence s with name "test"+testCounter. 
+ */
+  private void writeTest(PrintStream out, String className, int testCounter, ExecutableSequence s) {
+    out.println("  @Test");
+    out.println("  public void test" + testCounter + "() throws Throwable {");
+    out.println();
+    out.println(indent("if (debug) { System.out.println(); System.out.print(\"" + className + ".test" + testCounter + "\"); }"));
```

Comment:
This was a refactored method, so the code is at least partly what was already there. The use of indent is old code. I'm going to leave the inconsistency now, since formatting is on my "maybe" list.